src/index.js

```diff
@@ -5,12 +5,13 @@ import '@fortawesome/fontawesome-free/js/solid';
 import '@fortawesome/fontawesome-free/js/regular';
 import '@fortawesome/fontawesome-free/js/brands';
 
-import { add, edit, checkFocus } from './crud';
+import { add, edit, checkFocus, remove } from './crud';
+import { taskCompleted, clearCompleted, refresh } from './completed';
 import menuIcon from './menu-icon.png';
```

Comment:
- You did a great job in your importation but kindly make sure when you import the file you use the same name you give to that file your code has a few problems cause you changed the name of the file when imported kindle make sure you fix that. the file name have capital ` I `   `./menu-icon.png`  but when you import you used small `i` './menu-icon.png',  :+1:  :grinning: